src/test/article.spec.js

```diff
@@ -0,0 +1,158 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import 'chai/register-should';
+import 'chai/register-expect';
+import BaseRepository from '../repository/base.repository';
+import db from '../database/models';
+import app from '../index';
+import articleSample from '../database/mockdata/articledata';
+
+// configure chai to use expect
+chai.use(chaiHttp);
+const { expect } = chai;
+
+const articleEndpoint = '/api/v1/articles';
+
+let article;
+describe('Articles', () => {
+  const token =
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6NywidXNlcm5hbWUiOiJncmFjZSIsImVtYWlsIjoiZ3JlZW4xMjNAZ3JhY2UuY29tIiwicm9sZSI6InVzZXIiLCJpYXQiOjE1NjI3NDI3OTgsImV4cCI6MTU2MjgyOTE5OH0.mWSGB0rBbsy0YXcRBkyBF-DFos_guthFZK7Iauk8YJU';
+
+  beforeEach(async () => {
+    article = await BaseRepository.create(db.Article, articleSample);
+  });
+  afterEach(async () => {
+    await db.Article.destroy({ truncate: true, cascade: true });
+  });
+
+  describe('CREATE ARTICLE', () => {
+    it('should create article successfully, with valid user input', done => {
+      chai
+        .request(app)
+        .post(articleEndpoint)
+        .set('x-access-token', token)
+        .send(articleSample)
+        .end((error, response) => {
+          expect(response.status).to.equal(201);
+          done();
+        });
+    });
+  });
+
+  describe('GET ALL ARTICLES', () => {
+    before(() => {});
+
+    it('should fetch all successfully created articles', done => {
+      chai
+        .request(app)
+        .get(articleEndpoint)
+        .set('x-access-token', token)
+        .end((error, response) => {
+          expect(response.status).to.equal(200);
+          expect(response.body.data.length).to.be.greaterThan(0);
```

Comment:
You should test that the number of items returned matches what's in the `db`

Reply:
Thank you, work is in progress to implement the feedback received on this review.